src/main/java/com/icthh/xm/ms/entity/service/EventService.java

```diff
@@ -80,6 +84,18 @@ public Event saveEvent(Event event) {
         return permittedRepository.findAll(Event.class, privilegeKey);
     }
 
+    /**
+     * Get all the events by filter.
+     * Method is used in LEP's
+     *
+     * @return the list of entities
+     */
+    @Transactional(readOnly = true)
+    @SuppressWarnings("unused")
+    public List<Event> findAllByFilter(EventFilter filter) {
+        return eventQueryService.findAll(filter);
```

Comment:
what different with 
calendarService.findEvents
?

maybe dublicate?

Reply:
in "calendarService.findEvents" method calendarId - is required

and new method allow search event's without calendarId